<ui version="4.0" >
 <class>AutoPartWidget</class>
 <widget class="QWidget" name="AutoPartWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="1" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>408</width>
       <height>110</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="1" column="1" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Droid Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-size:12pt; font-weight:600;">Partitioning method&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignTop</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Please choose a partitioning method and then a device to partition.&lt;br>Device list will be updated based on the selected method.</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QRadioButton" name="accept_auto" >
          <property name="text" >
           <string>Automatic</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
          <property name="autoExclusive" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <layout class="QGridLayout" name="gridLayout" >
          <item rowspan="2" row="0" column="0" >
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" >
           <widget class="QRadioButton" name="accept_auto_1" >
            <property name="font" >
             <font/>
            </property>
            <property name="toolTip" >
             <string>This option will try to find free disk space of suitable size. If no such disk space exists, it will look for resizable partitions</string>
            </property>
            <property name="text" >
             <string>Try to use free space or resize current partitions</string>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="autoExclusive" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QRadioButton" name="accept_auto_2" >
            <property name="font" >
             <font/>
            </property>
            <property name="toolTip" >
             <string>This option will erase the entire disk. All the data on the disk will be lost.</string>
            </property>
            <property name="text" >
             <string>Delete all partitions</string>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="autoExclusive" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" >
         <widget class="QRadioButton" name="manual" >
          <property name="font" >
           <font/>
          </property>
          <property name="toolTip" >
           <string>This option allows partitioning the disk to your wishes.</string>
          </property>
          <property name="text" >
           <string>No, I'll partition my disk manually.</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="autoExclusive" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="textLabel1_2" >
         <property name="text" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Droid Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-size:12pt; font-weight:600;">Disk selection&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignTop</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="device_list" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="info" >
       <property name="text" >
        <string>&amp;nbsp;</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>17</width>
       <height>110</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>accept_auto</sender>
   <signal>clicked(bool)</signal>
   <receiver>accept_auto_1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>130</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel" >
     <x>160</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accept_auto</sender>
   <signal>clicked(bool)</signal>
   <receiver>accept_auto_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>127</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel" >
     <x>130</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manual</sender>
   <signal>clicked(bool)</signal>
   <receiver>accept_auto_1</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>102</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel" >
     <x>99</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manual</sender>
   <signal>clicked(bool)</signal>
   <receiver>accept_auto_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>204</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel" >
     <x>212</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
